packages/react/dismissable-layer/src/DismissableLayer.tsx

```diff
@@ -301,31 +302,37 @@ function useFocusOutside(onFocusOutside?: (event: FocusOutsideEvent) => void) {
  * -----------------------------------------------------------------------------------------------*/
 
 function createTotalLayerCount(displayName?: string) {
-  const TotalLayerCountContext = React.createContext<{
-    total: number;
-    setTotal: React.Dispatch<React.SetStateAction<number>>;
-  }>({ total: 0, setTotal: () => {} });
+  const [TotalLayerCountProviderImpl, useTotalLayerCountContext] = createContext(
```

Comment:
Doesn't matter for that change I guess but my brain still hurts thinking about whether this would need the same context treatment fix as the rest or not…